src/applications/travel-pay/components/complex-claims/pages/ConfirmationPage.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+
+import { TRAVEL_PAY_INFO_LINK } from '../../../constants';
+import { formatDateTime } from '../../../util/dates';
+
+const ConfirmationPage = () => {
+  // TODO: Remove placeholder data when wired up
+  const error = false;
+  const data = {
+    localStartTime: '2025-03-20T16:30:00.000-08:00',
+    location: {
+      attributes: {
+        name: 'Fort Collins VA Clinic',
+      },
+    },
+  };
+
+  const [formattedDate, formattedTime] = formatDateTime(data.localStartTime);
+
+  return (
+    <>
+      <h1>We’re processing your travel reimbursement claim</h1>
+      {error || !data ? (
+        <va-alert closable="false" status="error" visible>
+          <h2 slot="headline">Something went wrong on our end</h2>
+          <p className="vads-u-margin-y--0">
+            We’re sorry. We couldn’t file your travel reimbursement claim in
+            this tool right now. Please try again later.
+          </p>
+          <p>
+            Or you can still file for this appointment through the Beneficiary
+            Travel Self Service System (BTSSS).
+          </p>
+          <va-link
+            href={TRAVEL_PAY_INFO_LINK}
+            text="Find out how to file for travel reimbursement"
+          />
+        </va-alert>
+      ) : (
+        <va-alert status="success" visible>
+          <h2 slot="headline">Claim submitted</h2>
+          <p className="vads-u-margin-y--0">Claim number: #######</p>
+          <p>
+            This claim is for your appointment{' '}
+            {data.location?.attributes?.name
+              ? `at ${data.location.attributes.name}`
+              : ''}{' '}
+            {data.practitionerName ? `with ${data.practitionerName}` : ''} on{' '}
+            {formattedDate} at {formattedTime}.
+          </p>
+        </va-alert>
+      )}
+
+      <h2 className="vads-u-margin-top--4">Print this confirmation page</h2>
+      <p>
+        If you’d like to keep a copy of the information on this page, you can
+        print it now.
+      </p>
+      <va-button
+        text="Print this page for your records"
+        onClick={() => window.print()}
```

Comment:
What kind of print styling does this use? Are there default stylings? Do we need to use print stylings?